client/config/environment.js

```diff
@@ -17,6 +18,7 @@ module.exports = function (environment) {
     },
     NYC_ID_HOST,
     maintenanceTimes: getMaintenanceTimes(),
+    SHOW_ALERTS,
```

Comment:
My understanding is that the flag should resolve to a boolean value and it should be `false` by default.

From reading this, we take the environment string at face value without processing it into a boolean. When we use the value later in the code itself, we'll need to explicitly check its value every time

Reply:
Okay I updated it to use a function that returns a boolean depending on what it's set to like in the applicant portal PR. 

I notice we also have something similar in `config.services.ts` in applicant portal PR. Is `featureFlag` a more direct way of getting the feature flag env value when used in the server?

Reply:
> I notice we also have something similar in config.services.ts in applicant portal PR. Is featureFlag a more direct way of getting the feature flag env value when used in the server?

It is the way of using it in the server. Though, I think of it more in terms of "speaking the server's dialect" as opposed to "being more direct"